Actions/AddExistingApp/AddExistingApp.ps1

```diff
@@ -83,6 +83,7 @@ $telemetryScope = $null
 $bcContainerHelperPath = $null
 
 # IMPORTANT: No code that can fail should be outside the try/catch
+# IMPORTANT: All actions needs a try/catch here and not only in the yaml file, else they can silently fail
```

Comment:
That doesn't sound right.

The actions were probably silently failing because of the try/catch in the ps1 files: it catches the exception but doesn't throw is again. So, the outer try/catch (in the YAML files) never catches anything and, thus, never sets the exit code to 1.

Either one of the try/catch-es is redundant.

Reply:
Before this PR - there was no try/catch in the .ps1 - only in the yaml - and look at the run in the link - it gives an error in the .ps1 but the action succeeds.
The try/catch in the yaml will only catch if there is an error during the call (like the script doesn't exist)

Reply:
This comment is removed in the next checkin